manager/manager_test.go

```diff
@@ -177,6 +177,35 @@ func expectManagerWithContainersV2(containers []string, query *info.ContainerInf
 	return m, infosMap, handlerMap
 }
 
+func TestMachineInfoReturnValue(t *testing.T) {
```

Comment:
is this test from some other PR? or new code? In https://github.com/google/cadvisor/pull/2490/files I only see topology tests extended.

Reply:
Yeah, as you see in the commit log, it is is a separate test to avoid regressions. Open on splitting that to a separate PR or something, as it doesn't matter that much to me. All the tests have changed quite a bit, so it was impossible to cherry-pick back the test from the original PR.

We can just do a simple `copy := *m.machineInfo` instead of cherry-picking the commit, but I think @tedyu deserves the credit for the fix.